From 24341bbeaf54d69ca89de224c7bb75f45d5e7df9 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 9 Apr 2015 16:19:35 +0800
Subject: [PATCH] MLK-10629-3 dts: imx7d-sdb: add wifi-host property for
 usdhc2

There's a Broadcom WiFi on usdhc2.
Add this property due to Broadcom WiFi driver needs call wifi_card_detect
function.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 5f4985a..f08c2a2 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -893,6 +893,7 @@
 	keep-power-in-suspend;
 	tuning-step = <2>;
 	non-removable;
+	wifi-host;
 	status = "okay";
 };
 
-- 
1.8.0

